src/Bridge.ts

```diff
@@ -1142,7 +1142,7 @@ export class Bridge {
 
     private async setUpAdminRoom(roomId: string, accountData: AdminAccountData, notifContent: NotificationFilterStateContent) {
         const adminRoom = new AdminRoom(
-            roomId, accountData, notifContent, this.as.botIntent, this.tokenStore, this.config,
+            roomId, accountData, notifContent, this.as.botIntent, this.tokenStore, this.config, this.connectionManager!,
```

Comment:
Never use non-null here, always prefer a
```
if (!this.connectionManager) {
    throw Error('Expected connectionManager to be ready');
}
```

Reply:
Changed in b27e9657694b5ee09aa6b3537d76783a79745ee0.

One thing I often use instead in situations like this (with a lack of rxjs' `AsyncSubject`), is having this stored as a `Promise<ConnectionManager>`, initialized in the constructor and `await`ed everywhere it's needed – at the cost of some async-complexity this makes us immune to possible data races.

In this case however, it seems like ConnectionManager's async-ness is only there because of UserTokenStore needing a two-step initialization (`new` and `load()` later). Feels like we could easily avoid the problem by initializing both in the constructor, and hiding UTS' asyncness internally (since all its methods that require `load()` to complete first are `async` in the first place.

I'll draft us a way out of this, but maybe in another PR :)

Reply:
Yeah, promises are okay by me (if just a *little* ugly). And yeah, I loathe to have things which are possibly undefined after construction, so we should fix that.